frontend/src/components/worksheets/items/PlaceholderItem.js

```diff
@@ -58,7 +58,7 @@ export default forwardRef((props, ref) => {
     if (error) {
         return <div ref={ref}>Error loading item.</div>;
     }
-    if (item === null) {
+    if (!item) {
```

Comment:
Won't this change cause the placeholder to always say "No results found" (because `item` is initially undefined)? Let's change it back to

```suggestion
    if (item === null) {
```